README.md

```diff
@@ -31,7 +31,9 @@ Percona is dedicated to **keeping open source open**. Whenever possible, we stri
 
 ## How to get involved
 
-We encourage contributions and are always looking for new members that are as dedicated to serving the community as we are. The [Contributing Guide](https://github.com/percona/psmysql-docs/blob/innovation-release/contributing.md) contains how you can contribute.
+We encourage contributions and are always looking for new members that are as dedicated to serving the community as we are. The [Doc Contributing Guide](https://github.com/percona/psmysql-docs/blob/innovation-release/contributing.md) contains how you can contribute to the documentation.
+
+The [ CONTRIBUTING ](COTNRIBUTING.md) file in this directory containis instructions how to contribute to the code. 
```

Comment:
The CONTRIBUTING file in this directory contains instructions on contributing to the code.
not containis

Reply:
fixed